EncryptionBoxInput/EncryptionBoxInput.sol

```diff
@@ -0,0 +1,30 @@
+pragma ton-solidity >= 0.43.0;
+
+interface IEncryptionBoxInput {
```

Comment:
as @mikhailUshakoff rightly noted that we might want to have the `destroy` method in order to be able to kill the Encryption Box instance from debot once not needed anymore